src/main/resources/application.properties

```diff
@@ -32,6 +32,7 @@ javamelody.management-endpoint-monitoring-enabled=true
 # Health check settings
 health.databaseConnection.timeoutMillis=120000
 health.yarnConnection.testEndpoint=/cluster/cluster
+health.yarnConnection.timeoutMillis=120000
```

Comment:
Could you please add mention of this prop to the PR description? 
When I do release notes it is easier to scrap changes

Reply:
Done